packages/lighthouse-spa/src/app/dashboard/dashboard.service.spec.ts

```diff
@@ -1,9 +1,9 @@
-import { fakeAsync, flushMicrotasks, TestBed } from '@angular/core/testing';
+import { fakeAsync, flush, flushMicrotasks, TestBed } from '@angular/core/testing';
 import {
   ApolloTestingController,
   ApolloTestingModule,
 } from 'apollo-angular/testing';
-import { LeaderboardCategory } from 'app/leaderboard/enum';
```

Comment:
Hey @ayushsatyam146 can you roll back all these imports into absolute ones.

We have put baseURL in tsconfig so that these can be made into cleaner and easier imports

Reply:
I discussed this with @riginoommen. He said to keep it as it is because there are jest errors in the previous way.

Reply:
Kindly put it back. 

And set it in jest:https://stackoverflow.com/questions/50171412/jest-typescript-absolute-paths-baseurl-gives-error-cannot-find-module

This makes codebase import long and ugly. Kindly keep the codebase standards @riginoommen